src/structures/MessageAttachment.js

```diff
@@ -79,6 +79,12 @@ class MessageAttachment {
      * @type {?number}
      */
     this.width = typeof data.width !== 'undefined' ? data.width : null;
+
+    /**
+     * This media type of this attachment
+     * @type {?string}
+     */
+    this.contentType = typeof data.content_type !== 'undefined' ? data.content_type : null;
```

Comment:
```suggestion
    this.contentType = data.content_type ?? null;
```

Reply:
In theory I agree, but I kept it consistent with the rest of the file for now.